clients/src/main/java/org/apache/kafka/common/network/Selector.java

```diff
@@ -450,13 +450,24 @@ void pollSelectionKeys(Set<SelectionKey> selectionKeys,
                                 socketChannel.socket().getSendBufferSize(),
                                 socketChannel.socket().getSoTimeout(),
                                 channel.id());
+
+                        // Include PLAINTEXT in successful authentication metric, all others added after KafkaChannel.prepare()
+                        if (channel.ready())
+                            sensors.successfulAuthentication.record();
```

Comment:
Not sure about this one. It would basically be the same number as `connectionsCreated`. For `SSL` (even without authentication), it would still measure the cases where the connection is established and the handshake fails.

Reply:
Removed.